### PR TITLE
add resource for language sk-SK

### PR DESCRIPTION
**Pull request by klingger**
Add resource file for Slovak language

### CODE REVIEW

File: src/resources/sk-SK.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      return `nie je platná hodnota`;
+    },
+    'ContainsOnlyValidationRule': (newValue, threshold) => {
+      return `nie je platná hodnota`;
+    },
+    'StrongPasswordValidationRule': (newValue, threshold) => {
+      return `musí obsahovať kombináciu malých a veľkých písmen, číslic a špeciálnych znakov`;
+    },
+    'MediumPasswordValidationRule': (newValue, threshold) => {
+      return `should contain at least ${Utilities.getValue(threshold)} of the following groups: lowercase letters, uppercase letters, digits or special characters`;
```
Comment (Mordred) [inline src/resources/sk-SK.js]: Zruš tento return - Remove this return :)

**Comment by klingger** [comment]
Hotovo

Removed

**Comment by PWKad** [comment]
Hi @klingger thanks so much for this - can you please make sure the commit messages match the [contribution guide](https://github.com/DurandalProject/about/blob/master/CONTRIBUTING.md#commit) (its for changelog generation) and make sure to sign the CLA?

**Comment by klingger** [comment]
I'm already signed.

**Comment by PWKad** [comment]
Cool can you fix the commit messages?  If not when I do it when merging it looks like I actually made the contribution >.<

**Comment by PWKad** [comment]
@klingger I'm trying to merge but there are two new commits that don't match - any chance you can clean these up so I can merge in?

**Comment by klingger** [comment]
Sorry i am begginer in github and git, so can you explain me how this clean for merge?

**Comment by Gheoan** [comment]
@klingger from the command line:
 

- Change directory to the location of your local copy of https://github.com/klingger/validation
For example that directory for me would be `D:\username\projects\contrib\validation`
```shell
cd "D:\username\projects\contrib\validation"
```
If you don't know the location just clone the repository again:
```shell
git clone "https://github.com/klingger/validation.git" contrib-aurelia-validation
```
Then change directory to it:
```shell
cd contrib-aurelia-validation
```
- Now we want to "reset" the state of your local repository to how was it was before you started making changes without losing your changes:
```shell
git reset 425840323d4e41917ef83e3ef13dc653ad7bb489
```
425840323d4e41917ef83e3ef13dc653ad7bb489 is the last commit before you started making changes.
Now your changes to `src/resources/sk-SK.js` are untracked. If you run
```shell
git status
```
you will see `nothing added to commit but untracked files present (use "git add" to track)`
3. Let's add your changes to `src/resources/sk-SK.js` for commit:
```shell
git add *
```
`*` is for all untracked/changed/added files.
Now commit:
```shell
git commit -m "feat(I18N): add sk-SK"
```
`-m` commits with message without you using an editor
4. Now push the commits back to github
```shell
git push -f origin master
```
Done :smile: .

**Comment by PWKad** [comment]
Any update @klingger ?

**Comment by PWKad** [comment]
Closing until issues resolved.

**Pull request closed without merging**